lighthouse-core/fraggle-rock/gather/base-gatherer.js

```diff
@@ -90,6 +90,9 @@ class FRGatherer {
    * @return {Promise<LH.Gatherer.PhaseResultNonPromise>}
    */
   async afterPass(passContext, loadData) {
+    if ('dependencies' in this.meta) {
+      throw Error('Gatherer with dependencies should override afterPass');
```

Comment:
nice catch codecov! test for this in base-gatherer-test? :)

Reply:
Ah yes, test cases. I knew I was forgetting something.